source/form.html

```diff
@@ -293,87 +448,136 @@ <h2 class="add-plan__header">Добавить план:</h2>
                 <td class="alphabet-table__data">Я</td>
               </tr>
             </table>
-            <select class="select-country__countries country-list" name="country">
-              <option class="country-list__item" value="Австралия">
+            <select class="select-country__countries country-select-list" size="11" name="country-select">
+              <option class="country-select-list__item" value="Австралия">
                 Австралия
               </option>
-              <option class="country-list__item" value="Австрия">
+              <option class="country-select-list__item" value="Австрия">
                 Австрия
               </option>
-              <option class="country-list__item" value="Азербайджан">
+              <option class="country-select-list__item" value="Азербайджан">
                 Азербайджан
               </option>
-              <option class="country-list__item" value="Албания">
+              <option class="country-select-list__item" value="Албания">
                 Албания
               </option>
-              <option class="country-list__item" value="Алжир">
+              <option class="country-select-list__item" value="Алжир">
                 Алжир
               </option>
-              <option class="country-list__item" value="Ангола">
+              <option class="country-select-list__item" value="Ангола">
                 Ангола
               </option>
-              <option class="country-list__item" value="Андорра">
+              <option class="country-select-list__item" value="Андорра">
                 Андорра
               </option>
-              <option class="country-list__item" value="Анигуа и Барбуда">
+              <option class="country-select-list__item" value="Анигуа и Барбуда">
                 Анигуа и Барбуда
               </option>
-              <option class="country-list__item" value="Аргентина">
+              <option class="country-select-list__item" value="Аргентина">
                 Аргентина
               </option>
-              <option class="country-list__item" value="Армения">
+              <option class="country-select-list__item" value="Армения">
                 Армения
               </option>
-              <option class="country-list__item" value="Афганистан">
+              <option class="country-select-list__item" value="Афганистан">
                 Афганистан
               </option>
             </select>
           </div>
         </section>
-        <section class="add-plan__step-3 step-3 step">
-          <p class="step-3__title step__title">Шаг 3.<br>Развлечения</p>
-          <p class="step-3__title-descr step__title-descr">Наконец, расскажите о своих планах времяпровождения. Можно
+        <section class="add-plan__step-3 step-3">
+          <a name="step-3"></a>
+          <p class="step-3__title font-title-b-5">Шаг 3.<br>Развлечения</p>
```

Comment:
Нужно не ссылку ставить, а идентификатор на блок:
```suggestion
        <section class="add-plan__step-3 step-3" id="step-3">
          <p class="step-3__title font-title-b-5">Шаг 3.<br>Развлечения</p>
```